indico/modules/events/registration/models/forms.py

```diff
@@ -490,13 +491,13 @@ def get_personal_data_field_id(self, personal_data_type):
 
 @listens_for(orm.mapper, 'after_configured', once=True)
 def _mappers_configured():
-    query = (select([db.func.count(Registration.id)])
+    query = (select([db.func.coalesce(db.func.sum(Registration.occupied_slots), 0)])
```

Comment:
How can the sum be NULL? Shouldn't it at least be 1 for any registration? Or does this happen when there are no registrations yet?

Reply:
Yep, it's when there are no registrations